### ISSUE TITLE
[0.9.3.5 release-231] "weight by percent owned" checkbox in "property owned" does not weigh by percent owned

### ISSUE THREAD

**Issue by DanielKispert** [StrangeLoopGames/EcoIssues#22442]
1. have a property that is owned by a registrar title that is occupied by two people. In our example, the property is 12 plots big
![grafik](https://user-images.githubusercontent.com/34270661/124487659-487b7d80-ddaf-11eb-999d-e51b6f571927.png)
2. query the "property owned" of one of those occupants e.g. by law or executive action, like this:
![grafik](https://user-images.githubusercontent.com/34270661/124487797-6a750000-ddaf-11eb-9b39-e1696d2f156c.png)
3. result: 12 plots. Expected result: 6 plots (50% of 12 plots)
![grafik](https://user-images.githubusercontent.com/34270661/124487855-7b257600-ddaf-11eb-9044-06491fe3c414.png)

**Comment by SlayksWood** [comment]
@DanielKispert Thank you for the report.
It seems that we need a system like with bank accounts.
But I guess it will be stange to own 6,5 plots(if registrar title owns 13 plots)? or not?

**Comment by DanielKispert** [comment]
> @DanielKispert Thank you for the report.
> It seems that we need a system like with bank accounts.
> But I guess it will be stange to own 6,5 plots(if registrar title owns 13 plots)? or not?

Agreed, if like 5 people own a big property then there is a good argument to make that they all have access to all of it, so everyone owns 100% of it. If this option was removed and the "Property Owned" tooltip would say "If multiple people own a property, it counts as 100% for everyone", then the behavior would be clear as well.
If you do a system like with bank accounts, it may be worth having this flag be renamed to "divide ownership" or something like that, and have it off by default, so we can turn it on with the cogwheel if we need it (e.g. for a property tax)
